init.go

```diff
@@ -136,29 +135,31 @@ func (a *initFnCmd) init(c *cli.Context) error {
 		}
 	}
 
-	err = a.buildFuncFile(c)
+	err = a.buildFuncFile(c) // TODO: Return LangHelper here, then don't need to refind the helper in generateBoilerplate() below
 	if err != nil {
 		return err
 	}
 
-	runtimeSpecified := a.Runtime != ""
+	runtimeSpecified := a.ff.Runtime != ""
 
-	if runtimeSpecified && a.Runtime != funcfileDockerRuntime {
+	// TODO: why don't we treat "docker" runtime as just another language helper? Then can get rid of several Docker
```

Comment:
good point, been using `runtime: docker` for binaries, so it's really a nice-to-have thing.